jest.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  testEnvironment: 'node',
+  transform: {
+    "^.+\\.tsx?$": "ts-jest"
+  },
+  moduleFileExtensions: [
+    "ts",
+    "tsx",
+    "js",
+    "jsx",
+    "json",
+    "node",
+  ],
+  
+  testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.(ts|js)x?$',
+  coverageDirectory: 'coverage',
+  collectCoverageFrom: [
+    'src/**/*.{ts,tsx,js,jsx}',
+    '!src/**/*.d.ts',
+  ],
+  clearMocks: true,
+  modulePathIgnorePatterns: ['<rootDir>/build'],
+  coverageReporters: ["cobertura", "lcov", "text"],
```

Comment:
Lol, fun fact for you: `cobertura` means `coverage` in spanish

Reply:
Ha! Nice! I didn't know that.